lib/matplotlib/tests/test_ticker.py

```diff
@@ -239,6 +240,17 @@ def test_set_params(self):
         assert loc._base == 4
         assert list(loc._subs) == [2.0]
 
+    @image_comparison(['multiple_shared_log_axis.png'],
```

Comment:
What about checking the limits of your axes rather than an image comparison here?